InfiniteRecharge/src/main/java/frc/robot/subsystem/scoring/shooter/ShooterSubsystem.java

```diff
@@ -127,4 +132,23 @@ public void rotateToDeg(double targetPoint) {
         targetChange = 0;
     }
 
-}
\ No newline at end of file
+    public void rotateByDeg(double degrees) {
+        rotateToDeg(getTargetTurretDegGivenOffset(degrees));
+    }
+    
+    public double getTurretDeg() {
+        double encoderDeg =  azimuthMotor.getSelectedSensorPosition() * (360.0 / config.shooter.azimuth.ticksPerRevolution);

```

Comment:
yay math! for the sake of consistency though, we should use Elias's MathUtils.unitConverter. see usage on lines 121-123 for an example